charters/wg-2021-extension-plan.md

```diff
@@ -42,10 +42,11 @@ We are developing a new draft charter proposed to begin in 1 June 2023.
 | Mar 1, 2023 | Next | Next WG Charter - Final Draft - Submit |
 | Mar 15, 2023 | Profile | CR transition request |
 | Mar 30, 2023 | Profile | CR transition |
-| Mar 30, 2023 | Binding Templates | WG Note Publication |
+| Mar 30, 2023 | Binding Templates | Starting WG Review |
 | Mar 22, 2023 | Next | Next WG Charter AC Review Complete (submission + 1mo) |
```

Comment:
Note: There seem to be some issues with the dates ordering, see

Mar 22 and Mar 27, 2023 comes after Mar 30 etc.
Not your fault, but I think we should update it properly.